hugging_stanza.py

```diff
@@ -48,6 +57,23 @@ def write_model_card(repo_local_path, model):
     with open(readme_path, "w") as f:
         f.write(get_model_card(model))
 
+def get_model_tag(lang):
+    args = parse_args()
+    input_dir = args.input_dir
+
+    stanza.download(model_dir = input_dir, lang = lang)
+    model = Pipeline(model_dir= input_dir, lang=lang)
+    config_keys = model.config.keys()
+    tags = []
+    if "ner_model_path" in config_keys:
+        tags.append("ner")
+    if "pos_model_path" in config_keys:
+        tags.append("pos")
```

Comment:
These should be the full name based on https://github.com/huggingface/huggingface_hub/blob/main/widgets/src/lib/interfaces/Types.ts

`named-entity-recognition` and `part-of-speech-tagging`

Reply:
fixed 🤗